<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="MergeSortTest" tests="12" skipped="0" failures="0" errors="0" timestamp="2016-09-04T21:20:50" hostname="Shubham-PC" time="0.27">
  <properties/>
  <testcase name="test something basic" classname="MergeSortTest" time="0.08"/>
  <testcase name="in unroll : abc should raise exception" classname="MergeSortTest" time="0.079"/>
  <testcase name="in unroll : 2 should raise exception" classname="MergeSortTest" time="0.001"/>
  <testcase name="in unroll : true should raise exception" classname="MergeSortTest" time="0.0"/>
  <testcase name="in unroll : 4.6 should raise exception" classname="MergeSortTest" time="0.0"/>
  <testcase name="in unroll : [1, 2, 3] should raise exception" classname="MergeSortTest" time="0.029"/>
  <testcase name="in unroll : [1+i, 2, 3] should raise exception" classname="MergeSortTest" time="0.0"/>
  <testcase name="in unroll : [true, 2, 1] should raise exception" classname="MergeSortTest" time="0.001"/>
  <testcase name="in unroll : [2.4, 2+i, 1] should raise exception" classname="MergeSortTest" time="0.001"/>
  <testcase name="in unroll : Sorting [1, 4, 2] should give [1, 2, 4]" classname="MergeSortTest" time="0.043"/>
  <testcase name="in unroll : Sorting [4, 21, 5] should give [4, 5, 21]" classname="MergeSortTest" time="0.002"/>
  <testcase name="in unroll : Sorting [3.5, 1, 4] should give [1, 3.5, 4]" classname="MergeSortTest" time="0.002"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
